site/content/docs/user/ingress.md

```diff
@@ -84,6 +85,31 @@ Refer to [Using Ingress](#using-ingress) for a basic example usage.
 
 Additional information about Contour can be found at: [projectcontour.io](https://projectcontour.io)
 
+### Ambassador
```

Comment:
nit: I also meant move this entire section above contour too (alphabetical) 😬

Reply:
Sure, I can do that.